src/routes/safe/store/actions/transactions/__tests__/utils/transactionHelpers.test.ts

```diff
@@ -0,0 +1,875 @@
+import { getMockedSafeInstance, getMockedTxServiceModel } from 'src/test/utils/safeHelper'
```

Comment:
This file tests functions imported from `src/logic/safe` but placed in `src/routes/safe`